js/src/client.ts

```diff
@@ -1150,33 +1150,51 @@ export class Client {
 
   private async _sendMultipartRequest(parts: MultipartPart[], context: string) {
     try {
-      const formData = new FormData();
+      // Create multipart form data manually using Blobs
+      const boundary =
+        "----LangSmithFormBoundary" + Math.random().toString(36).slice(2);
+      const chunks: Blob[] = [];
+
       for (const part of parts) {
-        formData.append(part.name, part.payload);
+        // Add field boundary
+        chunks.push(new Blob([`--${boundary}\r\n`]));
+        chunks.push(
+          new Blob([
+            `Content-Disposition: form-data; name="${part.name}"\r\n`,
+            `Content-Type: ${part.payload.type}\r\n\r\n`,
```

Comment:
Missing content length for each part

Reply:
No it's set above:

https://github.com/langchain-ai/langsmith-sdk/pull/1235/files#diff-a4957c11dcb337079c4682e6d0069736d061422f086fea164792577b977e7149R1100

Reply:
Ah got it